yarn.lock

```diff
@@ -0,0 +1,895 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Remove. Lock files should only be for apps. We want to know if we break because of a dependency.